### ISSUE TITLE
UserWarning for boltzdesign.py

### ISSUE THREAD

**Issue by Maikuraky** [yehlincho/BoltzDesign1#27]
When running boltzdesign.py, this UserWarning is prompted. Do I need to change it and which version should I change it to? 
Thank you for your reply
`./BoltzDesign1/boltzdesign/boltzdesign_utils.py:599: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.detach().clone() or sourceTensor.detach().clone().requires_grad_(True), rather than torch.tensor(sourceTensor).
  return grad * torch.sqrt(torch.tensor(eff_L)) / (gn + 1e-7)`
